base/pkg.jl

```diff
@@ -13,9 +13,17 @@ type PkgError <: Exception
     msg::AbstractString
 end
 
-for file in split("dir types reqs cache read query resolve write entry git")
-    include("pkg/$file.jl")
-end
+include("pkg/dir.jl" )
```

Comment:
I think the loop is more in keeping with [the DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) (Don't Repeat Yourself). Using `split` with a space-delimited string is just more convenient than writing out `["dir","types",...]`.

I personally prefer how it is, but anyway it wasn't intended to be a criticism so much as just wondering what your motivation was for changing it. (Also my opinion doesn't really matter; I'm just some guy.)